contracts/zero-ex/tests/MetaTransactionV2Test.t.sol

```diff
@@ -0,0 +1,457 @@
+// SPDX-License-Identifier: Apache-2.0
+/*
+  Copyright 2023 ZeroEx Intl.
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+*/
+
+pragma solidity ^0.6.5;
+pragma experimental ABIEncoderV2;
+
+import "./utils/BaseTest.sol";
+import "forge-std/Test.sol";
+import "./utils/LocalTest.sol";
+import "../contracts/src/features/MetaTransactionsFeatureV2.sol";
+import "../contracts/src/features/interfaces/IMetaTransactionsFeatureV2.sol";
+import "../contracts/src/features/interfaces/IMetaTransactionsFeature.sol";
+import "../contracts/test/TestMintTokenERC20Transformer.sol";
+import "../contracts/src/features/libs/LibSignature.sol";
+import "src/features/libs/LibNativeOrder.sol";
+import "../contracts/test/tokens/TestMintableERC20Token.sol";
+import "@0x/contracts-utils/contracts/src/v06/errors/LibRichErrorsV06.sol";
+import "@0x/contracts-erc20/src/IEtherToken.sol";
+
+contract MetaTransactionTest is LocalTest {
+    address private constant ZERO_ADDRESS = address(0);
```

Comment:
Why is this a named constant?

Reply:
Just copy/pasted from previous test code that we wrote. I can get rid of it.